ironfish/src/fileStores/hostsStore.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { FileSystem } from '../fileSystems'
+import { PeerAddress } from '../network/peers/peerAddress'
+import { KeyStore } from './keyStore'
+
+export type HostsOptions = {
+  priorConnectedPeers: PeerAddress[]
```

Comment:
NIT: I think the word `Connected` indicates `Prior` already. What do you think about simplifying this name?

Reply:
How about `priorPeers`?